Cargo.toml

```
@@ -18,7 +18,7 @@ circle-ci = { repository = "kyren/hashlink", branch = "master" }
 serde_impl = ["serde"]
 
 [dependencies]
-hashbrown = { version = "0.15", default-features = false, features = ["default-hasher", "inline-more"] }
+hashbrown = { version = "0.16", default-features = false, features = ["default-hasher", "inline-more"] }
```

If we could make this a range dependency instead of a hard upgrade, that would make my life for upgrading `hashbrown` incrementally in Firefox a lot easier! 😖🙏🏻

---

For convenience, that would make this line:

```suggestion
hashbrown = { version = ">=0.15,<=0.16", default-features = false, features = ["default-hasher", "inline-more"] }
```